test_web/index.html

```diff
@@ -50,16 +50,27 @@
             </script>
     </nav>
     <div id="test-container">
-        <div class="test-wrapper" id="add-cookie">
+        <div class="test-wrapper" id="headerChanger">
             <div class="form-group">
-                <label for="cookie">Cookie 🍪</label> <span id="cookie-content"></span> 
-                <input type="email" class="form-control" id="cookie" aria-describedby="cookieHelp" placeholder="cf_clearance=abcxyzblalba">
-                <small id="cookieHelp" class="form-text text-muted">Add cookie into headers to bypass cloudflare</small>
-                <button type="button" class="btn btn-primary" id="add-cookie-btn">Add</button>
-            </div>
+                <label for="cookie">Headers</label>
+                <div id="headerinfos">
+                    <div class="headerinfo" data-headername="User-Agent">
+                        <span class="headername" >User-Agent</span><b>:</b>
+                        <span class="headervalue"></span>
+                    </div>
+                    <div class="headerinfo">
+                        <span class="headername">Cookie</span><b>:&nbsp;</b>
+                        <span class="headervalue" contenteditable="true" ></span>
+                    </div>
+                </div>
+                <button id="addheaderbtn" class="btn-secondary pl-1 pr-1">Add header</button>
+            </div> 
```

Comment:
I think only dynamic header `cookie`  should be customizable. Other headers like referer, encoding,... should be added manually in plugin code by contributors.